apps/web/components/booking/AvailableEventLocations.tsx

```diff
@@ -8,54 +8,61 @@ import { FiLink } from "@calcom/ui/components/icon";
 
 import type { Props } from "./pages/AvailabilityPage";
 
+const excludeNullValues = (value: unknown) => !!value;
+
 export function AvailableEventLocations({ locations }: { locations: Props["eventType"]["locations"] }) {
```

Comment:
_Organizer's default app_ is not shown as a location in the booking description. I have _Google Meet_ and _Organizer's default app_ set as locations: 
<img width="282" alt="Screenshot 2023-03-20 at 11 34 02" src="https://user-images.githubusercontent.com/30310907/226315289-7d46ed85-910c-4c04-9b4a-dbadb622afff.png">

Everything else looks good to me 👍

Reply:
I do this intentionally - Wouldn't make much sense to a booker to talk about organizers on a round robin event, e.g. Ideally we'd already know who you will be booked with at the booking stage but that's a future improvement 🙏